kodilla-jdbc/src/test/java/com/kodilla/jdbc/StoredProcTestSuite.java

```diff
@@ -27,10 +27,36 @@ public void testUpdateVipLevel() throws SQLException {
         String sqlCheckTable = "SELECT COUNT(*) AS HOW_MANY FROM READERS WHERE VIP_LEVEL=\"Not set\"";
         ResultSet rs = statement.executeQuery(sqlCheckTable);
         int howMany = -1;
-        if (rs.next()){
+        if (rs.next()) {
             howMany = rs.getInt("HOW_MANY");
 
-            Assert.assertEquals(0,howMany);
+            Assert.assertEquals(0, howMany);
+        }
+    }
+
+    @Test
+    public void testUpdateBestsellers() throws SQLException {
+
+        //given
+        DbManager dbManager = DbManager.getInstance();
+        String sqlUpdate = "UPDATE BOOKS SET BESTSELLER=\"9\"";
+        Statement statement = dbManager.getConnection().createStatement();
+        statement.executeUpdate(sqlUpdate);
+
+        //when
+
+        String sqlProcedureCall = "CALL updateBestseller()";
+        statement.execute(sqlProcedureCall);
+        //then
+
+        String sqlCheckTable = "SELECT COUNT(*) AS HOW_MANY FROM READERS WHERE VIP_LEVEL=\"9\"";
```

Comment:
I don't understand this test. Correct me if I am wrong, but updateBestseller updated BOOKS table.
But you check READERS table, hmm...

Reply:
Of course i made a mistake. It was late.. 
I've just corrected sql query.